docs/storefront/graphql/examples/products.mdx

```diff
@@ -87,6 +87,27 @@ Query all products by not including an argument for `products`.
 
 You can also query for featured products, related products, and more. See the [GraphQL Storefront Playground](https://developer.bigcommerce.com/graphql-storefront/playground) for full schema documentation.
 
+### Get a product by URL using a route node
+
+Query a list of products by matching the `Redirect` type. 
```

Comment:
You can only query a single resource (product in this case) at a time with the `route` node.